css/style.css

```diff
@@ -309,50 +312,308 @@
 
 .red-first {
   position: absolute;
-  transform: translate(226%, -185px);
+  transform: translate(227%, -186px);
   background-color: #e51937;
   color: #fff;
   max-width: 441px;
   height: 900px;
   box-sizing: border-box;
-  padding: 54px 0 0 55px;
+  padding: 53px 0 0 52px;
 }
 
 .description__title {
   margin: 0 auto;
   font-weight: 500;
-  font-size: 31px;
+  font-size: 33px;
   text-align: left;
   white-space: nowrap;
-  letter-spacing: 2px;
+  letter-spacing: 2.5px;
+  padding-bottom: 8px;
 }
 
-.description__text {
+.drive {
   margin: 0 auto;
   display: block;
   max-width: 100%;
   font-size: 16px;
-  padding: 11px 47px 71px 0;
+  padding: 4px 71px 69px 0;
   letter-spacing: 1px;
 }
 
 .description__screen {
   display: flex;
-  padding-bottom: 51px;
+  padding-bottom: 40px;
+}
+
+.img-inch {
+  padding-right: 24px;
 }
 
 .displ {
   display: flex;
-  flex-wrap: nowrap;
+  flex-direction: column;
+}
+
+.data__figures {
+  font-size: 31px;
+  padding-right: 10px;
+}
+
+.data__ins {
+  font-size: 21px;
+  letter-spacing: 1px;
 }
 
 .data__display {
-  white-space: nowrap;
-  display: inline-block;
+  font-size: 12px;
+  letter-spacing: 1px;
+  width: 100%;
 }
 
 .description__wifi {
-  padding-bottom: 38px;
+  display: flex;
+  padding-bottom: 35px;
+}
+
+.img-wifi {
+  width: 40px;
+  height: 32px;
+  padding: 13px 17px 0 0;
+}
+
+.wifi-text {
+  display: flex;
+  flex-direction: column;
+}
+
+.update {
+  display: flex;
+  flex-direction: column;
+}
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)